Update balena.yml

mpous opened the pull request:
Update description and post-provisioning

mpous committed:
Update balena.yml

Update description and post-provisioning

Bucknalla commented:
Hey @mpous :) Thanks for the suggestion. What is the purpose of this correction?

mpous commented:
Hey @Bucknalla actually i wanted to fix the description of the block, as there is a link on markdown that is not properly visualized and it's too long as description. 

<img width="343" alt="Captura de pantalla 2023-07-21 a les 18 20 13" src="https://github.com/arribada/nanomq-balena/assets/173156/55614e84-a147-4ccf-81ca-c55b86a185ad">

So i moved the current description as post-provision and i added a shorter description. Not relevant, but i guess it will be visualized in a better way :) 

Let me know if it makes sense!

Bucknalla reviewed (approved):
Looks good to me @mpous thanks :)
